src/app/components/Filters/FieldsetList.jsx

```diff
@@ -56,21 +56,28 @@ class FieldsetList extends React.Component {
         {legend && <legend><h3>{legend}</h3></legend>}
         <ul className="nypl-generic-checkbox nypl-generic-columns">
           {
-            values.map((filter, i) => (
-              <li className="nypl-generic-checkbox" key={i}>
-                <input
-                  id={`${filter.label}-label`}
-                  type="checkbox"
-                  name="filters"
-                  value={JSON.stringify(_extend({ field: filterId }, filter))}
-                  onClick={e => this.onFilterClick(e, filter)}
-                  checked={filter.selected}
-                />
-                <label htmlFor={`${filter.label}-label`}>
-                  {filter.label} ({filter.count.toLocaleString()})
-                </label>
-              </li>
-            ))
+            values.map((filter) => {
+              let filterLabel = filter.label;
+              if (filter.label === 'Greek, Modern (1453- )') {
+                filterLabel = 'Greek, Modern (1453-present)';
```

Comment:
Are more mappings going to be added for the "present" designation? Would this warrant expression matching to add the substituted phrase?

Reply:
:( This should be the only one. This eventually should be coming from the API and not the front-end, but I'd say mapping on the front-end is fine.